target/linux/mpc85xx/base-files/etc/board.d/02_network

```diff
@@ -32,6 +32,9 @@ tplink,tl-wdr4900-v1)
 watchguard,firebox-t10)
 	ucidef_set_interfaces_lan_wan "eth1 eth2" "eth0"
 	;;
+watchguard,xtm330)
+	ucidef_set_interfaces_lan_wan "port1 port2 port3 port4 port5 port6" "port0"
```

Comment:
I guess the network isn't working in failsafe mode. Consider modification of the [05_set_preinit_iface_mpc85xx ](https://github.com/openwrt/openwrt/blob/main/target/linux/mpc85xx/base-files/lib/preinit/05_set_preinit_iface_mpc85xx)file.

Reply:
Looks fine. You can mark the conversations as resolved if GitHub permits this :)